charts/nexus-repository-manager/README.md

```diff
@@ -84,15 +84,15 @@ To install the chart, use the following:
 $ helm install nexus-rm sonatype/nexus-repository-manager [ --version v29.2.0 ]
 ```
 
-The above command deploys Nexus on the Kubernetes cluster in the default configuration.
+The above command deploys Nexus Repository on the Kubernetes cluster in the default configuration.
 
 You can pass custom configuration values as follows:
 
 ```bash
 $ helm install -f myvalues.yaml sonatype-nexus ./
 ```
 
-The default login is randomized and can be found in sonatype /nexus-data/admin.password
+The default login is randomized and can be found in sonatype/nexus-data/admin.password
```

Comment:
sonatype/nexus-data/admin.password may also benefit from `markup` but I'd need to run the env to be sure.

Reply:
actually, the path is just /nexus-data/admin.password, so you can drop "sonatype" completely.

Reply:
^^ fix that, and i'll merge this PR. thanks.

Reply:
Ah I see, maybe it meant Sonatype's directory_path and I misread.  I will fix tho :)

Reply:
668e2a3